.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+- repo: https://github.com/ambv/black
+  rev: 20.8b1
+  hooks:
+  - id: black
+- repo: https://github.com/asottile/reorder_python_imports
+  rev: v2.3.5
+  hooks:
+  -   id: reorder-python-imports
+- repo: https://gitlab.com/pycqa/flake8
+  rev: '3.8.4'
+  hooks:
+  - id: flake8
+    args: ["--config=setup.cfg"]
```

Comment:
Is this required? flake8 supports setup.cfg as one of its default config files.

Reply:
Hmm true, I'll remove it, thanks.